build.sbt

```diff
@@ -82,16 +82,19 @@ def getSparkVersion(): String = {
     "latest",
     SPARK_MASTER_VERSION,
     LATEST_RELEASED_SPARK_VERSION,
-    latestReleasedSparkVersionShort
+    latestReleasedSparkVersionShort,
+    "4.0.0-preview1"
   )
 
   // e.g. build/sbt -DsparkVersion=master, build/sbt -DsparkVersion=4.0.0-SNAPSHOT
-  val input = sys.props.getOrElse("sparkVersion", LATEST_RELEASED_SPARK_VERSION)
+  val input = sys.props.getOrElse("sparkVersion", defaultSparkVersion)
```

Comment:
make this `"4.0.0-preview1"` be the "default" spark version that is used when you do NOT provide `-DsparkVersion`